### PR TITLE
Vish/Assignment-6/assignmentA6_1.html and style.css Uploading

### PR DESCRIPTION
**Pull request by VIP5614**

### CODE REVIEW

**Review by mayank30041995 — APPROVED** [review]

**Review by sanketmod — APPROVED** [review]

**Review by poonam-khode — APPROVED** [review]

File: Vish/assignment-6/VishnuA6_1.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Document</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <div class="box">
+            <h2>MI-9 Intern HTML Assesment</h2>
+        </div>
+        <hr>
+        <form action="#">
```
Comment (MKadeer17) [inline Vish/assignment-6/VishnuA6_1.html]: I don't Think we need to define action attribute if there is no submit button 
.

File: Vish/assignment-6/VishnuA6_1.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            <div class="box">
+                <h3>Personal Details</h3>
+            </div>
+            <div>
+                <span>First Name:</span>
```
Comment (MKadeer17) [inline Vish/assignment-6/VishnuA6_1.html]: Here label  tag is use not span.

File: Vish/assignment-6/VishnuA6_1.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        <form action="#">
+            <div class="box">
+                <h3>Personal Details</h3>
+            </div>
+            <div>
+                <span>First Name:</span> 
+                <input type="text" placeholder="First Name">
+            </div> 
+            <div>
+                <span>Last Name:</span>
```
Comment (MKadeer17) [inline Vish/assignment-6/VishnuA6_1.html]: Same mistake here also add label not span.

File: Vish/assignment-6/VishnuA6_1.html [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+            <div>
+                <span>First Name:</span> 
+                <input type="text" placeholder="First Name">
+            </div> 
+            <div>
+                <span>Last Name:</span> 
+                <input type="text" placeholder="Last Name">
+            </div> 
+            <div>
+                <span>Gender:</span>
```
Comment (MKadeer17) [inline Vish/assignment-6/VishnuA6_1.html]: Same here also

File: Vish/assignment-6/VishnuA6_1.html [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+                <span>First Name:</span> 
+                <input type="text" placeholder="First Name">
+            </div> 
+            <div>
+                <span>Last Name:</span> 
+                <input type="text" placeholder="Last Name">
+            </div> 
+            <div>
+                <span>Gender:</span> 
+                <input type="radio"> <label for="Male">Male</label>
```
Comment (MKadeer17) [inline Vish/assignment-6/VishnuA6_1.html]: input tag add id attribute for matching label for attribute.

File: Vish/assignment-6/VishnuA6_1.html [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+                <input type="radio"> <label for="Male">Female</label>
```
Comment (MKadeer17) [inline Vish/assignment-6/VishnuA6_1.html]: Same here

File: Vish/assignment-6/VishnuA6_1.html [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+                <input type="radio"> <label for="Male">Other</label>
```
Comment (MKadeer17) [inline Vish/assignment-6/VishnuA6_1.html]: Same here

File: Vish/assignment-6/VishnuA6_1.html [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+            </div> 
+            <div>
+                <span>Age:</span>
```
Comment (MKadeer17) [inline Vish/assignment-6/VishnuA6_1.html]: You should use label tag not span tag both are different things

File: Vish/assignment-6/VishnuA6_1.html [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+            </div> 
+            <div>
+                <span>Gender:</span> 
+                <input type="radio"> <label for="Male">Male</label>
+                <input type="radio"> <label for="Male">Female</label>
+                <input type="radio"> <label for="Male">Other</label>
+            </div> 
+            <div>
+                <span>Age:</span> 
+                <input type="radio"> <label for="less then 18">less then 18</label>
```
Comment (MKadeer17) [inline Vish/assignment-6/VishnuA6_1.html:35]: Here you also not use the id attribute in input tag.

File: Vish/assignment-6/VishnuA6_1.html [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+                <input type="radio" checked> <label for="18-25">18-25</label>
```
Comment (MKadeer17) [inline Vish/assignment-6/VishnuA6_1.html:36]: Same here please use id attribute input type radio , matching with label for attribute

File: Vish/assignment-6/VishnuA6_1.html [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+                <input type="radio"> <label for="26-40">26-40</label>
```
Comment (MKadeer17) [inline Vish/assignment-6/VishnuA6_1.html:37]: Same here please use id attribute input type radio , matching with label for attribute

File: Vish/assignment-6/VishnuA6_1.html [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
+            </div> 
+            <div class="box">
+                <h3>Academic Details</h3>
+            </div>
+            <div>
+                <span>Skill Set:</span>
```
Comment (MKadeer17) [inline Vish/assignment-6/VishnuA6_1.html]: use Label tag not span

File: Vish/assignment-6/style.css [hunk 13]
```diff
@@ -0,0 +1,20 @@
...
+.box{
+    background-color: #8a9ba6;
+    border-radius:0 10px 0 10px;
+    margin: 20px 0 10px 0;
+
+}
+hr{
+    margin-top: 50px;
+}
+div span{
```
Comment (MKadeer17) [inline Vish/assignment-6/style.css]: use span only for font-weight not div, because in div tag you not write any text but you write text in heading tag.

**Review by MKadeer17 — COMMENTED** [review]
Please do those changes

File: Vish/assignment-6/VishnuA6_1.html (hunk 1 above)
Comment (VIP5614) [reply Vish/assignment-6/VishnuA6_1.html]: ok

File: Vish/assignment-6/VishnuA6_1.html (hunk 2 above)
Comment (VIP5614) [reply Vish/assignment-6/VishnuA6_1.html]: ok

File: Vish/assignment-6/VishnuA6_1.html (hunk 3 above)
Comment (VIP5614) [reply Vish/assignment-6/VishnuA6_1.html]: ok

File: Vish/assignment-6/VishnuA6_1.html (hunk 5 above)
Comment (VIP5614) [reply Vish/assignment-6/VishnuA6_1.html]: ok

File: Vish/assignment-6/VishnuA6_1.html (hunk 8 above)
Comment (VIP5614) [reply Vish/assignment-6/VishnuA6_1.html]: ok

File: Vish/assignment-6/VishnuA6_1.html (hunk 10 above)
Comment (VIP5614) [reply Vish/assignment-6/VishnuA6_1.html:36]: ok

File: Vish/assignment-6/VishnuA6_1.html (hunk 11 above)
Comment (VIP5614) [reply Vish/assignment-6/VishnuA6_1.html:37]: ok

File: Vish/assignment-6/VishnuA6_1.html (hunk 12 above)
Comment (VIP5614) [reply Vish/assignment-6/VishnuA6_1.html]: ok

**Review by MKadeer17 — COMMENTED** [review]
Good.

**Review by MKadeer17 — APPROVED** [review]

**Pull request merged**